.travis.yml

```diff
@@ -33,6 +33,29 @@ jobs:
           on:
             repo: open-policy-agent/gatekeeper
             tags: true # only push release image tag for tags
+    - stage: "helm build, test"
+      script:
+        - make manager
+        - make e2e-bootstrap
+        - export KUBECONFIG="$(kind get kubeconfig-path --name="kind")"
+        - make e2e-build-load-image IMG=gatekeeper-e2e:latest
+        - make e2e-helm-deploy HELM_REPO=gatekeeper-e2e HELM_RELEASE=latest
+        - make test-e2e
+        - echo -e '\n\n======= manager logs =======\n\n' && kubectl logs -n gatekeeper-system gatekeeper-controller-manager-0 manager
+      deploy:
```

Comment:
We should have `deploy` steps for only one stage... likely not this one.

Reply:
Good catch - removed